modules/virt-vm-storage-volume-types.adoc

```diff
@@ -30,6 +30,10 @@ PVC.
 DataVolumes build on the *persistentVolumeClaim* disk type by managing the process
 of preparing the virtual machine disk via an import, clone, or upload operation.
 VMs that use this volume type are guaranteed not to start until the volume is ready.
++
+Define DataVolumes with a `type` of *dataVolume*. If another volume type is
+used in error, such as *PersistentVolumeClaim*, a warning will be triggered
```

Comment:
I'm not sure for this reference, why are the type name start with lowercase？Is that because YAML is case sensitive and they are like these on the YAML? If so, here *PersistentVolumeClaim* should be *persistentVolumeClaim*.

Reply:
note yaml is case sensitive, and when kubernetes parses it, and if the case is wrong it will ignore the field and not parse it.

Reply:
Thanks @awels